src/components/EditorSideBar/EditorSideBarContent.tsx

```diff
@@ -2,6 +2,7 @@ import { Box } from "@chakra-ui/react";
 import React from "react";
 import { ChevronLeftIcon } from "@chakra-ui/icons";
 import Blueprints from "./Blueprints";
+import Folder from "./Folder";
```

Comment:
Is this "Folder" on SideBar? If so, should the path be /components/EditorSideBar/Folder?